lib/generate.js

```diff
@@ -35,36 +37,64 @@ function dedentFunc(func) {
     });
 }
 
+function tabulateString(string, indent, filter) {
+    if(Array.isArray(string)) {
+      string = string.join('\n');
+    }
+    tabulated = string.toString().split('\n').map(function addIndent(line, i) {
```

Comment:
Seems inefficient, you're `join()`ing the array and then `split()`ing it again. :-/